src/rstudio/__init__.py

```diff
@@ -36,7 +36,7 @@ def send_to_rstudio(cmd):
             time.sleep(0.001)
 
             if not control_was_released:
-                win32api.keybd_event(win32con.VK_CONTROL, 0, 0, 0)
+                win32api.keybd_event(win32con.VK_CONTROL, 0, win32con.KEYEVENTF_KEYUP, 0)
```

Comment:
This line should simulate the control keydown as line [34](https://github.com/TimothyHyndman/SendCode/blob/3be4906c55f22e1283352bfd10ed4dd516e49182/src/rstudio/__init__.py#L34) lifted up the control key. This modification (or removing this line) will always send "Enter" instead of "Ctrl+Enter". However, it doesn't allow doing "Ctrl + Enter" multiple times by holding "Ctrl" key and pressing "Enter" multiple times.